### ISSUE TITLE
Formatted text showing up in memo/journal

### ISSUE THREAD

**Issue by joehays** [ccbogel/QualCoder#758]
I copy pasted some formatted text into a memo/journal entry and the formatting stayed. This surprised me since everything in QualCoder is raw text-based. There was no way to remove the formatting. Not sure if you just want to leave this or not. It's not a big deal but was an unexpected behavior. 

To be honest, I'd like basic formatting abilities (e.g. basic markdown) in all the text documents but not sure if that's in line with your vision for the tool.

**Comment by ccbogel** [comment]
Yes, I can't fix that. It is an issue that has been raised before. I have considered applying some markdown - headings and italics and bold maybe. with the markdown text showing showing - easier to implement for me
